bolt/lib/Rewrite/BinaryPassManager.cpp

```diff
@@ -154,6 +155,11 @@ static cl::opt<bool>
                       cl::desc("print profile quality/bias analysis"),
                       cl::cat(BoltCategory));
 
+static cl::opt<bool>
+    PrintContinuityStats("print-continuity-stats",
+                      cl::desc("print profile function CFG continuity stats"),
+                      cl::init(true), cl::cat(BoltCategory));
```

Comment:
I'd rather leave the pass off by default, in line with other profile analyses/reports available.